packages/api/es/indexer.js

```diff
@@ -403,35 +535,49 @@ function logHandler(event, context, cb) {
   });
 }
 
+/**
+ * Lambda function handler for sns2elasticsearch
+ *
+ * @param  {Object} event - incoming message sns
+ * @param  {Object} context - aws lambda context object
+ * @param  {function} cb - aws lambda callback function
+ * @returns {Promise} undefined
+ */
 function handler(event, context, cb) {
   // we can handle both incoming message from SNS as well as direct payload
   log.debug(JSON.stringify(event));
   const records = get(event, 'Records');
   let jobs = [];
 
   if (records) {
-    jobs = records.map(r => handlePayload(r));
+    jobs = records.map(handlePayload);
   }
   else {
     jobs.push(handlePayload(event));
   }
 
-  Promise.all(jobs).then(r => {
+  return Promise.all(jobs).then((r) => {
```

Comment:
Most of our other code has `.then()` and `.catch()` starting on new lines, indented from the previous line.

```
return Promise.all(jobs)
  .then((r) => {
    blah
  })
  .catch(cb);
```